README.md

```diff
@@ -43,36 +44,40 @@ First, make sure that you have the following dependencies:
 
 Then, run the following commands to clone and build SourceCred:
 
-```
-git clone https://github.com/sourcecred/sourcecred.git
-cd sourcecred
-yarn install
-yarn backend
-export SOURCECRED_GITHUB_TOKEN=YOUR_GITHUB_TOKEN
-node bin/sourcecred.js load REPO_OWNER/REPO_NAME
-# this loads sourcecred data for a particular repository
-# it can take a few mins to run and will exit when finished
-yarn start
-# then navigate to localhost:8080 in your browser
+```Bash
+$ git clone https://github.com/sourcecred/sourcecred.git
```

Comment:
The downside to including the prompt tokens is that you can no longer
copy and paste the sequence of commands into their shell. Omitting the
prompt is therefore pretty common (at a glance: [Flow], [TensorBoard],
[TensorFlow], and [fzf] all do, and those are the first that I tried).
Suggest reverting this change.

[fzf]: https://github.com/junegunn/fzf#using-homebrew-or-linuxbrew
[Flow]: https://github.com/facebook/flow#installing-flow-per-project
[TensorBoard]: https://github.com/tensorflow/tensorboard#usage
[TensorFlow]: https://github.com/tensorflow/tensorflow#installation

Reply:
Good call. Just getting them into a `bash` block will look better than what we had which was really all I was trying to do there.

Reply:
I think the intent was to remove the `$` tokens, but that revert doesn't seem to have made it into the change.